fts-cron/renew_fts_proxy_multi_vo.sh.j2

```diff
@@ -1,36 +1,41 @@
 #!/bin/bash
 
-{% if RUCIO_FTS_VOMSES is defined %}
-{% set vomses = RUCIO_FTS_VOMSES.split(',') %}
-{% for voms in vomses %}
+{% if RUCIO_FTS_VOS is defined -%}
+{% set vos = RUCIO_FTS_VOS.split(' ') %}
+{% for vo in vos %}
 
-printf "=================== {{ voms }} Renewal =========================\n\n"
+printf "=================== {{ vo }} Renewal =========================\n\n"
 
 # We have to copy the certificates because we cannot change permissions on them as mounted secrets and voms-proxy is particular about permissions
-mkdir /tmp/{{ voms }}/
-cp /opt/rucio/certs/{{ voms }}/{{ RUCIO_LONG_PROXY }} /tmp/{{ voms }}/long.proxy
-chmod 400 /tmp/{{ voms }}/long.proxy
+mkdir /tmp/{{ vo }}/
+cp /opt/rucio/certs/{{ vo }}/{{ RUCIO_LONG_PROXY }} /tmp/{{ vo }}/long.proxy
+chmod 400 /tmp/{{ vo }}/long.proxy
 
 # Generate a proxy with the voms extension if requested
-voms-proxy-init2 -valid 24:00 -cert /tmp/{{ voms }}/long.proxy -key /tmp/{{ voms }}/long.proxy -out /tmp/x509up_{{ voms }} -voms {{ voms }} -rfc -timeout 5
+if [ $RUCIO_FTS_VOMS_{{ vo | upper }} ]
+then
+    voms-proxy-init2 -valid 24:00 -cert /tmp/{{ vo }}/long.proxy -key /tmp/{{ vo }}/long.proxy -out /tmp/x509up_{{ vo }} -rfc -timeout 5 -voms $RUCIO_FTS_VOMS_{{ vo | upper }}
```

Comment:
perhaps the outputs here should also be to the VO directory, and then use a generic name for the output, but organised by the directories, rather than prescribing the naming of the files?

Reply:
I agree, makes it a bit easier to understand and more consistant